enterprise/management/commands/create_enterprise_course_enrollments.py

```diff
@@ -39,50 +42,81 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         enterprise_customer_uuid_filter = options.get('enterprise_customer_uuid')
         records_created = 0
+        records_failed = 0
 
         missing_enrollment_data = self._fetch_course_enrollment_data(
             enterprise_customer_uuid_filter
         )
         for item in missing_enrollment_data:
-            enterprise_customer_uuid = item['enterprise_customer_uuid']
+            does_course_exist_in_catalog = False
             user_id = item['user_id']
             course_run_id = item['course_run_id']
+            enterprise_customer_uuid = item['enterprise_customer_uuid']
 
             # pylint: disable=no-member
             enterprise_customer = EnterpriseCustomer.objects.get(uuid=enterprise_customer_uuid)
-            if enterprise_customer.catalog_contains_course(course_run_id):
-                enterprise_customer_user = EnterpriseCustomerUser.objects.get(
+
+            try:
+                does_course_exist_in_catalog = enterprise_customer.catalog_contains_course(course_run_id)
+            except (SlumberBaseException, ConnectionError, Timeout) as exc:
```

Comment:
This exception handling needs to done [here](https://github.com/edx/edx-enterprise/blob/dc274195f43fb118c1b7482acc94e5b6fe4c9020/enterprise/models.py#L1536) to cover larger use cases